### PR TITLE
POC/Provisioning: Use cards layout to show more status info

### PR DESCRIPTION
**Pull request by ryantxu**
This updates the repository list to show more details in the Cards metadata:

image

Specifically, it now shows the github repository and link

Compared to our current list view 😬 
image

### CODE REVIEW

File: pkg/registry/apis/provisioning/repository_controller.go [hunk 1]
```diff
@@ -232,12 +232,12 @@ func (rc *RepositoryController) sync(key string) error {
 		if cachedRepo.Status.Initialized {
 			status, err = repo.OnUpdate(ctx, logger)
 			if err != nil {
-				return fmt.Errorf("on create repository: %w", err)
+				rc.logger.Error("OnUpdate", "error", err)
```
Comment (ryantxu) [inline pkg/registry/apis/provisioning/repository_controller.go:235]: @MissingRoberto -- returning an error here makes an inf loop!

File: public/app/features/provisioning/RepositoryListPage.tsx [hunk 2]
```diff
@@ -51,36 +64,73 @@ function RepositoryListPageContent({ items }: { items?: Repository[] }) {
...
-                          <DeleteRepositoryButton name={name} />
-                        </div>
-                      </>
-                    </Alert>
-                  </>}
-                </Stack>
+                  )}
+                  {healthy && <SyncRepository repository={item} />}
+                </Card.Actions>
+                <Card.SecondaryActions>
```
Comment (ryantxu) [inline public/app/features/provisioning/RepositoryListPage.tsx:129]: https://developers.grafana.com/ui/latest/index.html?path=/story/general-card--full

File: public/app/features/provisioning/RepositoryListPage.tsx [hunk 3]
```diff
@@ -51,36 +64,73 @@ function RepositoryListPageContent({ items }: { items?: Repository[] }) {
...
+                              <br />
+                              <br />
+                            </div>
+                          ))}
+                        ></Alert>
+                      )}
+                    </>
+                  ) : (
+                    <div>
+                      <Alert severity="warning" title="repository initializing" />
```
Comment (ryantxu) [inline public/app/features/provisioning/RepositoryListPage.tsx:113]: for the couple seconds after creating a repository, it will not yet have a status

Comment (ryantxu) [reply public/app/features/provisioning/RepositoryListPage.tsx:113]: ideally this page is watching the repositorys and keeping them up-to-date, so would re-render on change

**Review by Clarity-89 — APPROVED** [review]
🎉

**Pull request merged**